shuup/core/migrations/0099_move_settings_to_db.py

```diff
@@ -0,0 +1,83 @@
+from django.conf import settings
+from django.db import migrations
+
+from shuup import configuration
+from shuup.admin.setting_keys import (
+    SHUUP_ADMIN_ALLOW_HTML_IN_PRODUCT_DESCRIPTION,
+    SHUUP_ADMIN_ALLOW_HTML_IN_VENDOR_DESCRIPTION,
+)
+from shuup.core.setting_keys import (
+    SHUUP_ADDRESS_HOME_COUNTRY,
+    SHUUP_ALLOW_ANONYMOUS_ORDERS,
+    SHUUP_ALLOW_ARBITRARY_REFUNDS,
+    SHUUP_ALLOW_EDITING_ORDER,
+    SHUUP_ALLOWED_UPLOAD_EXTENSIONS,
+    SHUUP_CALCULATE_TAXES_AUTOMATICALLY_IF_POSSIBLE,
+    SHUUP_DEFAULT_ORDER_LABEL,
+    SHUUP_DISCOUNT_MODULES,
+    SHUUP_ENABLE_ATTRIBUTES,
+    SHUUP_ENABLE_MULTIPLE_SHOPS,
+    SHUUP_ENABLE_MULTIPLE_SUPPLIERS,
+    SHUUP_HOME_CURRENCY,
+    SHUUP_LENGTH_UNIT,
+    SHUUP_MANAGE_CONTACTS_PER_SHOP,
+    SHUUP_MASS_UNIT,
+    SHUUP_MAX_UPLOAD_SIZE,
+    SHUUP_ORDER_SOURCE_MODIFIER_MODULES,
+    SHUUP_PRICING_MODULE,
+    SHUUP_REFERENCE_NUMBER_LENGTH,
+    SHUUP_REFERENCE_NUMBER_METHOD,
+    SHUUP_REFERENCE_NUMBER_PREFIX,
+    SHUUP_TAX_MODULE,
+    SHUUP_TELEMETRY_ENABLED,
+    SHUUP_VOLUME_UNIT,
+)
+from shuup.front.setting_keys import SHUUP_FRONT_MAX_UPLOAD_SIZE
+from shuup.reports.setting_keys import DEFAULT_REPORTS_ITEM_LIMIT
+
+
+def move_settings_to_db(apps, schema_editor):
+    configuration.set(None, SHUUP_HOME_CURRENCY, settings.SHUUP_HOME_CURRENCY)
```

Comment:
for every setting, use a getattr and fallback to the value that was in settings.py:

```py
configuration.set(None, SHUUP_ALLOW_ANONYMOUS_ORDERS, getattr(settings, "SHUUP_ALLOW_ANONYMOUS_ORDERS", True))
```

We need to do this because we need to delete the entries from the settings.py, we shouldn't keep those anymore.